spring-amqp/src/main/java/org/springframework/amqp/core/AsyncAmqpTemplate.java

```diff
@@ -115,12 +115,12 @@
 	 * If the post processor adds a correlationId property, it must be unique.
 	 * @param exchange the exchange
 	 * @param routingKey the routing key.
-	 * @param message the message.
+	 * @param object the object to convert.
 	 * @param messagePostProcessor the post processor.
 	 * @param <C> the expected result type.
 	 * @return the {@link AmqpConverterFuture}.
 	 */
-	<C> AmqpConverterFuture<C> convertSendAndReceive(String exchange, String routingKey, Object message,
+	<C> AmqpConverterFuture<C> convertSendAndReceive(String exchange, String routingKey, Object object,
```

Comment:
Recall me, please, why do we need these two `ListenableFuture` extensions without any customization and so on.
I mean `AmqpConverterFuture`  and `AmqpMessageFuture`
Thanks